packages/mobile/src/identity/verification.ts

```diff
@@ -549,52 +626,83 @@ function* isCodeAlreadyAccepted(code: string) {
   return existingCodes.find((c) => c.code === code)
 }
 
-function* revealNeededAttestations(
+function* revealAttestations(
   attestationsWrapper: AttestationsWrapper,
   account: string,
   phoneHashDetails: PhoneNumberHashDetails,
-  attestations: ActionableAttestation[],
-  withoutRevealing: boolean = false
+  attestations: ActionableAttestation[]
 ) {
-  Logger.debug(TAG + '@revealNeededAttestations', `Revealing ${attestations.length} attestations`)
-  const delayPeriod = 5000
+  Logger.debug(TAG + '@revealAttestations', `Revealing ${attestations.length} attestations`)
   let i = 0
+  const delayPeriod = 5000
+  const reveals = []
+  for (const attestation of attestations) {
+    const result = yield call(
```

Comment:
Nit: if I understand correctly, `result` is always a boolean. If so, consider renaming to `success` because `result` implies an outcome and makes me think it's either some data or `undefined`